js/about/newTabComponents/clock.js

```diff
@@ -11,14 +11,26 @@ class Clock extends React.Component {
     this.state = this.getClockState(new Date())
   }
   get formattedTime () {
-    const time = this.state.currentTime
-    return time.replace(' AM', '').replace(' PM', '')
+    return this.state.currentTime.map(component => {
```

Comment:
nice cleanup here 😄

Reply:
It was mainly so I could vertically center the ':' character in the time. I just wasn't sure if it existed in all locales, so I did this...